modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -7,26 +7,22 @@ endif::[]
 :imagesdir: ../assets/images
 
 You can see a history of changes you make to a project in Flow Designer.
```

Comment:
```suggestion
You can see a history of changes you make to a project in Flow Designer. This enables you to revert to a previous version of your project, should you need to.
```

Reply:
I just added the second part of that sentence to provide a link to the sudden comment about "restore a saved state" that comes later. I guessed. Adjust as necessary to retain intended tech meaning.

Reply:
In my view, I think the fact that users can revert is clear from the rest of the doc.